rest_api/app/workers/payment_worker.rb

```diff
@@ -0,0 +1,24 @@
+require "json"
+
+class PaymentWorker
+  include Sneakers::Worker
+  from_queue "payment_update"
+
+  def work(msg)
+    puts "::::::::::: PAYMENT RECEIVED ::::::::::"
+    policy_data = JSON.parse(msg, symbolize_names: true)
+
+    ActiveRecord::Base.connection_pool.with_connection do
+      policy = Policy.find_by(payment_id: policy_data[:payment_id])
+      policy.update!(status: "active")
```

Comment:
eu vi que no model de policy você colocou o status como `enum`, o active record criar uns métodos baseados no seu enum, então você pode atualizar a apolice chamando assim
```suggestion
      policy.active!
```